### ISSUE TITLE
No navbar on 404 page

### ISSUE THREAD

**Issue by Isak-Kallini** [Dsek-LTH/web#401]
#399 changed the error page a bit. It works for all errors except 404's, which get caught by the error page outside the `(app)` group. (It does still work if the 404 is on a sub route though, for example `/events/abc`)

**Comment by Macludde** [comment]
Yeah I don't know how to fix this. Previously the layout was duplicated, once in `+layout.svelte` and once in `+error.svelte`, which I really didn't like. I tried to move the `+error.svelte` into the `(app)` folder but of course, then it would not catch anything at the top level. I guess one solution would be to create a "Layout.svelte" component which both `+layout.svelte` and `+error.svelte` can use.

Like this:
```html
<Layout>
  <slot />
 </Layout>
``` 

 The problem is also, the layout requires certain data, data which is fetched inside `(app)/+layout.server.ts`, that would then have to be moved to `+layout.server.ts`

**Comment by Macludde** [comment]
This can be handled like it is handled in the #366 branch. We create a `[...path]` route which just throws a 404 like this:
```typescript
// [...path]/+page.server.ts
import { error } from "@sveltejs/kit";
import type { PageServerLoad } from "./$types";

/**
 * This route catches all requests that
 * don't match any other /nolla routes.
 *
 * It returns a 404 error that will be
 * handled by the /nolla/+error page,
 * instead of the root +error page.
 */

export const load: PageServerLoad = async () => {
  error(404);
};
```

**Comment by Macludde** [comment]
This should be fixed, can't remember which PR but I think #416
